src/Http/Middleware/SecurityHeadersMiddleware.php

```diff
@@ -115,15 +184,15 @@ public function setXFrameOptions($option = 'sameorigin', $url = null)
      * @param string $mode Mode value. Available Values: '1', '0', 'block'
      * @return $this
      */
-    public function setXssProtection($mode = 'block')
+    public function setXssProtection($mode = self::XSS_BLOCK)
     {
         $mode = (string)$mode;
 
-        if ($mode === 'block') {
-            $mode = '1; mode=block';
+        if ($mode === self::XSS_BLOCK) {
+            $mode = self::XSS_ENABLED_BLOCK;
```

Comment:
XSS_BLOCK and XSS_ENABLED_BLOCK could be merged to single constant, as only `'1; mode=block'` is real option, and `'block'` is just shortcut. Constant would hide complexity of real string.

Reply:
@garas while I agree about hiding complexity, ``block`` is currently used a few lines above in the method signature. https://github.com/cakephp/cakephp/pull/12664/files/ec711f5b99384897f594c03a41c6ceae21f063f4#diff-660cb26ba064296df9cc50f62b2d9455R187

Reply:
Default argument value also should be changed to constant of full string.

In this case, new default will not change result of method.

For BC, literal `'block'` should be used in `if` condition